### PR TITLE
Refactor eacruntime yaml to efcruntime yaml

### LINKED ISSUE #3122: [FEATURES]: refactor eacruntime yaml to efcruntime yaml
**What feature you'd like to add:**
refactor eacruntime yaml to efcruntime yaml

**Why is this feature needed:**

### PR DESCRIPTION
**Pull request by yangjun289519474**
### Ⅰ. Describe what this PR does
Refactor eacruntime yaml to efcruntime yaml

### Ⅱ. Does this pull request fix one issue?

fixes #3122 

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: pkg/ddc/efc/transform_image_test.go [hunk 1]
```diff
@@ -41,7 +41,7 @@ func TestParseFuseImage(t *testing.T) {
 func TestParseWorkerImage(t *testing.T) {
 	engine := &EFCEngine{}
 	image, tag, imagePullPolicy := engine.parseWorkerImage("", "", "")
-	if image != "registry.cn-zhangjiakou.aliyuncs.com/nascache/eac-worker" ||
+	if image != "registry.cn-zhangjiakou.aliyuncs.com/nascache/efc-worker" ||
 		tag != "update" || imagePullPolicy != "IfNotPresent" {
```
Comment (TrafalgarZZZ) [inline pkg/ddc/efc/transform_image_test.go:45]: Should change `tag != "update"` to `tag != "latest"` to pass the unit tests

**Review by TrafalgarZZZ — CHANGES_REQUESTED** [review]

File: pkg/ddc/efc/transform_image_test.go (hunk 1 above)
Comment (yangjun289519474) [reply pkg/ddc/efc/transform_image_test.go]: fix

**Review by cheyang — APPROVED** [review]
/lgtm
/approve

**Review by TrafalgarZZZ — APPROVED** [review]
/lgtm

**Pull request merged**